src/js/common/components/Challenge/ThanksForViewingChallenge.jsx

```diff
@@ -0,0 +1,130 @@
+import { IconButton } from '@mui/material';
+import { Close } from '@mui/icons-material';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Confetti from 'react-confetti';
```

Comment:
Thank you Erin! Please make sure to put react-confetti into /Users/dalemcgrew/PycharmProjects/WebApp/package.json